posthog/migrations/0314_team_extra_settings.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-05-09 19:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Based on the PG migration happening on the 12th lets not ship this before that.

Reply:
why? how would this affect things?

Reply:
the last PG migration failed, because there was a django migration that broke replication